src/holofuel/pages/TransactionHistory/TransactionHistory.js

```diff
@@ -118,7 +125,7 @@ export default function TransactionsHistory ({ history: { push } }) {
       <div styleName='balance-amount'>
         <DisplayBalance
           holofuelBalance={holofuelBalance}
-          ledgerLoading={ledgerLoading} />
+          ledgerLoading={!hasloadedFirstLedger && ledgerLoading} />
```

Comment:
this can be 
`ledgerLoading={!holofuelBalance && ledgerLoading}  />` 
and then you can remove lines 50 - 55